src-db/database/model/tables/ETDEP_DEPENDENCY.xml

```diff
@@ -49,10 +49,18 @@
         <default/>
         <onCreateDefault/>
       </column>
+      <column name="VERSION_STATUS" primaryKey="false" required="false" type="VARCHAR" size="60" autoIncrement="false">
+        <default/>
+        <onCreateDefault/>
+      </column>
       <column name="UNINSTALLDEPENDENCY" primaryKey="false" required="true" type="CHAR" size="1" autoIncrement="false">
         <default><![CDATA[N]]></default>
         <onCreateDefault><![CDATA['N']]></onCreateDefault>
       </column>
+      <column name="CHANGE_VERSION" primaryKey="false" required="true" type="CHAR" size="1" autoIncrement="false">
```

Comment:
You must create a check to ensure that this check field accept only 'Y' or 'N' values. See core checks.

Also add checks for anohter check fields (ISACTIVE and UNINSTALLDEPENDENCY).

Reply:
Also add checks for anohter check fields (ISACTIVE and UNINSTALLDEPENDENCY).